docs/API/get-started.md

```diff
@@ -0,0 +1,37 @@
+Title: Get Started with Flotiq API Integration
+
+Description: Get started with integrating Flotiq API into your application.
+
+!!! note "Before you begin, make sure you have the following:"
+    * Flotiq Account: Sign up for an account on Flotiq website if you haven't already.
+    * API Key: Obtain your Application Read and Write API Key from the Flotiq dashboard. You'll need it to perform API operations.
+    * Basic understanding of RESTful APIs: Familiarize yourself with the concepts of RESTful APIs if you're new to this.
```

Comment:
Always link to related pages and doc articles. Make sure to open external pages (like flotiq registration page) in a new tab.

Reply:
Also, I think for a quick start, we should ask user to model their API in the dashboard. I'd imagine a general use case for getting started as:
1. User registers through webpage
2. User creates their CTD and a couple of COs
3. User queries data using API
4. User pushes data using API

Because of this, I'd rewrite the docs to assume that the user has CTD and COs and focus on the meat - data management in Flotiq.